src/content/getting-started/architecture/networkplugin-syncer/_index.en.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Networkplugin syncer"
+---
+
+The **_networkplugin-syncer_** interfaces to the K8s network plugin subsystem
+in any necessary way. Currently, it's only deployed for _OVNKubernetes_,
+see the [OVNKubernetes architecture section](./ovn-kubernetes/) for more details.
```

Comment:
I know but that's an implementation detail. The bottomline is that the CNI plugin is detected - the fact that it's passed in is detecting it.

Reply:
We should use "CNI network plugin" or "network plugin" according to the the core k8s people, otherwise they complain when you do that :D

I will rephrase all that, but I want to keep the keep to the architecture section.